config/routes.rb

```diff
@@ -135,12 +135,12 @@
   match '/@:username/a/:id', to: 'answer#show', via: 'get', as: :show_user_answer_alt
   match '/@:username/q/:id', to: 'question#show', via: 'get', as: :show_user_question_alt
   match '/@:username/followers(/p/:page)', to: 'user#followers', via: 'get', as: :show_user_followers_alt, defaults: {page: 1}
-  match '/@:username/friends(/p/:page)', to: 'user#friends', via: 'get', as: :show_user_friends_alt, defaults: {page: 1}
+  match '/@:username/followings(/p/:page)', to: 'user#followings', via: 'get', as: :show_user_followings_alt, defaults: {page: 1}
```

Comment:
should we make a redirect from the old `/@:username/friends` url to the new one?